app/controllers/githook_controller.rb

```diff
@@ -43,4 +46,9 @@ def authenticate
   rescue ActionController::RedirectBackError
     redirect_to root_path
   end
+
+  def verify_signature(payload_body)
+      signature = 'sha1=' + OpenSSL::HMAC.hexdigest(OpenSSL::Digest.new('sha1'), ENV['SECRET_TOKEN'], payload_body)
```

Comment:
Line is too long. [115/80]<br>Use 2 (not 4) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.